CONTRIBUTING.md

```diff
@@ -25,131 +156,92 @@ sudo apt install socat
    cd binderhub
    ```
 
-1. [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to run Kubernetes locally.
+1. [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+   to run Kubernetes locally.
 
-   For MacOS, you may find installing from https://github.com/kubernetes/minikube/releases may be
-   more stable than using Homebrew.
+   For MacOS, you may find installing from https://github.com/kubernetes/minikube/releases
+   may be more stable than using Homebrew.
```

Comment:
I'm not certain that this statement is true anymore. Certainly when I first tried the contributing guidelines I had a lot of minikube-related issues that were fixed by installing via Homebrew.

Reply:
Deleted.